src/FourBitTwoDisclosureDeviceUnlocker.java

```diff
@@ -104,7 +104,9 @@ public static boolean unlock(final Device dev) {
                 n--;
             }
         }
-        if (isUnlocked) appendTrace("device is unlocked");
+        if (isUnlocked) {
```

Comment:
Suggested change was to make this adhere to agreed upon code style for readability.